src/components/GestureHandlerButton.tsx

```diff
@@ -1,10 +1,5 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-import { HostComponent, requireNativeComponent } from 'react-native';
+import { HostComponent } from 'react-native';
 import { RawButtonProps } from './GestureButtons';
-import { shouldUseCodegenNativeComponent } from '../utils';
-
-const RNGestureHandlerButtonNativeComponent = shouldUseCodegenNativeComponent()
-  ? require('../fabric/RNGestureHandlerButtonNativeComponent').default
-  : requireNativeComponent('RNGestureHandlerButton');
+import RNGestureHandlerButtonNativeComponent from '../fabric/RNGestureHandlerButtonNativeComponent';
```

Comment:
just wondering if the directory name `fabric` still makes sense

Reply:
Yeah, I guess renaming it to `codegen` would be a good idea.

Reply:
in many places i've seen name `components` which kind of makes sense but in our case `src/components/components` would be weird. as for `codegen`, it's ok but it's not true for Paper (although `codegenNativeComponent` is also a scam), but maybe `native` instead?

Reply:
One more thing that comes to mind is `specs`, but if not then `native` sounds good.

Reply:
we have a winner here